packages/edit-site/src/components/dataviews-actions/index.js

```diff
@@ -21,8 +22,20 @@ export const useEditPostAction = () => {
 			label: __( 'Edit' ),
 			isPrimary: true,
 			icon: edit,
-			isEligible( { status } ) {
-				return status !== 'trash';
+			isEligible( post ) {
+				if ( post.status === 'trash' ) {
+					return false;
+				}
+				// It's eligible for all post types except patterns.
+				if (
+					! [ ...Object.values( PATTERN_TYPES ) ].includes(
```

Comment:
Can't we just merge the two conditions into one. `post.type !== PATTERN_TYPES.theme`

Reply:
Of course 😄